tests/acceptance/reminder-list-test.js

```diff
@@ -13,12 +13,12 @@ test('viewing the homepage', function(assert) {
   visit('/');
 
   andThen(function() {
-    assert.equal(currentURL(), '/');
+    assert.equal(currentURL(), '/reminders');
     assert.equal(Ember.$('.spec-reminder-item').length, 5);
```

Comment:
We assumed it was necessary for those tests because they were already written for us. We deleted it to just do a straight equal and then a find and everything still passes. I would assume it's not necessary because we're using qUnit and we can just do regular assert.equal.